src/test/java/pages/BasePage.java

```diff
@@ -0,0 +1,138 @@
+package pages;
```

Comment:
Is there any reason why `BasePage` is in `test/java`, not `main/java`?

Reply:
My mistake. I fixed it.